.vscode/settings.json

```diff
@@ -1,5 +1,8 @@
 {
   "typescript.tsdk": "node_modules/typescript/lib",
   "typescript.enablePromptUseWorkspaceTsdk": true,
-  "svg.preview.background": "custom"
+  "svg.preview.background": "custom",
+  "cSpell.words": [
+    "Cloudagent"
+  ]
 }
```

Comment:
Does this needs to be included or something .gitignore?

Reply:
removed